_tutorial/wire_gen.go

```diff
@@ -0,0 +1,18 @@
+// Code generated by Wire. DO NOT EDIT.
+
+//go:generate wire
+//+build !wireinject
+
+package main
+
+// Injectors from wire.go:
+
+func InitializeEvent(phrase string) (Event, error) {
```

Comment:
This is what this repository's Wire is producing. The comments are not copied over because injectors don't copy that part of the AST. Opened #87 to track that.